integrations/extensions/docs/elasticsearch-install-and-setup/how_to_use_web_crawler_in_elasticsearch.md

```diff
@@ -0,0 +1,315 @@
+    ```shell
+    ES_URL=<your-elasticsearch-host>
+    ES_USER=<your-elasticsearch-username>
+    ES_PASSWORD=<your-elasticsearch-password>
+    
+    ES_VERSION=<your-elasticsearch-version>
+    ES_CACERT_FOLDER=<your-config-folder-containing-elasticsearch-cert>
+    ES_CACERT_NAME=<your-es-cert-file-name>
+    ```
+  NOTE: `ES_CACERT_FOLDER` can be your kibana config folder if you have stored the elasticsearch certificate in the folder following [ICD-elasticsearch-install-and-setup](../../docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md)
+
+* Download and start Enterprise Search
+    ```shell
+    docker run \
+    --name "enterprise-search" \
+    --network "elastic" \
+    --publish "3002:3002" \
+    -v "${ES_CACERT_FOLDER}:/usr/share/enterprise-search/es-config:ro" \
+    --interactive \
+    --tty \
+    --rm \
+    --env "secret_management.encryption_keys=[${ENCRYPT_KEY}]" \
+    --env "allow_es_settings_modification=true" \
+    --env "elasticsearch.host=${ES_URL}" \
+    --env "elasticsearch.username=${ES_USER}" \
+    --env "elasticsearch.password=${ES_PASSWORD}" \
+    --env "elasticsearch.ssl.enabled=true" \
+    --env "elasticsearch.ssl.certificate_authority=/usr/share/enterprise-search/es-config/${ES_CACERT_NAME}" \
+    --env "kibana.external_url=http://kibana:5601" \
+    "docker.elastic.co/enterprise-search/enterprise-search:${ES_VERSION}"
+    ```
+* Verify it 
+  * Open http://localhost:5601 in your browser and log into Kibana using your Elasticsearch username and password.
+  * Navigate to the Search Overview page http://localhost:5601/app/enterprise_search/overview.
+  * If you can see `Web Crawler` available as an option to ingest content, your Enterprise Search has been set up successfully.
+
+### Set up Enterprise Search for watsonx Discovery on CloudPak
+Coming soon...
+
+## Step 2: Create and configure a web crawler in Elasticsearch 
+* In Kibana, you can find the Web Crawler option on Search Overview http://localhost:5601/app/enterprise_search/overview. 
+Click on `Start`, and follow the steps to create a Web Crawler index. 
+
+
+* On the `Manage Domains` tab, add a domain, for example, `https://www.nationalparks.org`.
+
+
+* Add an entry point under the domain `Entry points` tab, for example, `https://www.nationalparks.org/explore/parks`
+
+
+* From the domain `Crawl Rules` tab, add two additional rules like below:  
+  <img src="assets/add_crawl_rules_for_web_crawler.png" width="802" height="309" />  
+  NOTE: The two rules tell the web crawler to only crawl URLs that begin with https://www.nationalparks.org/explore/parks. 
+  Learn more about crawl rules from [here](https://www.elastic.co/guide/en/app-search/8.10/web-crawler-reference.html#web-crawler-reference-crawl-rule)
+
+
+* You can now try the web crawler by clicking on `Crawl` in the upper right corner.  
+  NOTE: The web crawler uses a default ingest pipeline to extract web content and store them as documents in the index. 
```

Comment:
I don't understand this note.  Are you saying users should do step 3 before step 2?  If so, why is step 2 before step 3?

Reply:
Step 2 is about creating a web crawler, and step 3 is about creating a custom ingest pipeline for the web crawler. So, step 3 is a more advanced step and it should happen after step 2. 

This note here informs the users that they can try starting their web crawler now, but it will be using a default ingest pipeline without chunking and ELSER. To enable chunking and ELSER, they need to finish step 3 and start the web crawler again. 

And then at the end of step 3, we instruct that you need to delete the existing documents before starting the web crawler again with the custom ingest pipeline. 
https://github.com/watson-developer-cloud/assistant-toolkit/pull/245/files#diff-245b7192fa6a5e6c3adf79907d12fcdc1131088813a49a9fb391bb60f930174eR261-R271

Reply:
Thanks, Bill. I updated it as you suggested. I also repurposed the `delete_all_docs` query at the end of step 3. https://github.com/watson-developer-cloud/assistant-toolkit/pull/245/files#diff-245b7192fa6a5e6c3adf79907d12fcdc1131088813a49a9fb391bb60f930174eR278-R289. It should be more clear now.